packages/nuxt/src/app/composables/url.ts

```diff
@@ -0,0 +1,11 @@
+import { getRequestURL as getServerRequestURL } from 'h3'
+import { useRequestEvent } from './ssr'
+import { useRuntimeConfig } from '#app'
+
+export function getRequestURL () {
```

Comment:
Note: This might conflict with server util types accepting `event` as the first param (unless we introduce `server/tsconfig.ts` in same minor release of nuxt to avoid

Reply:
Since this is a composable (with context) `use` makes sense to me

Reply:
Makes sense to add `server/tsconfig.json` in next minor.